registry/handler/handler.go

```diff
@@ -26,10 +27,15 @@ func (r *Registry) GetService(ctx context.Context, req *pb.GetRequest, rsp *pb.G
 }
 
 func (r *Registry) Register(ctx context.Context, req *pb.Service, rsp *pb.EmptyResponse) error {
-	err := r.Registry.Register(service.ToService(req))
+	var regOpts []registry.RegisterOption
+	ttl := time.Duration(req.Options.Ttl)
```

Comment:
Does this imply TTL will be sent as nanoseconds? We're using seconds everywhere else so we may want to process this as seconds.

Reply:
It should not matter in the grand scheme of things. Because 0.0 is 0.0 if either of nano or just seconds is used. But let's be consistent and change it so seconds 👍